packages/indexer-common/src/allocations/keys.ts

```diff
@@ -38,6 +42,11 @@ export const allocationSigner = (wallet: Wallet, allocation: Allocation): string
   )
 }
 
+// Returns allocation signer wallet
+export const allocationSigner = (wallet: Wallet, allocation: Allocation): Signer => {
```

Comment:
Yeah looks like this was added afterwards, and decided to use the same naming 
One only returns the privateKey and the other the wallet
If you want to change the name its okay, but I think its okay
Since it returns what it is
the allocationSigner and the allocationSignerPrivateKey